### ISSUE TITLE
Default value doesn't populate on 'New Media' Modal dialog box.

### ISSUE THREAD

**Issue by MinSomai** [payloadcms/payload#1039]
# Bug Report

Default value for relationship field not populated. It is working when uploading media from the Media collection itself but not when uploading from other collection.

## Expected Behavior
Works correctly in the Media collection. The 'owner' field.

![working](https://user-images.githubusercontent.com/21338691/187735458-2db4fec6-0458-49b4-84a4-f5d65fe0f938.png)

## Current Behavior
`owner` field's default value is empty. And because it is readOnly, we cannot change it.

![readonly](https://user-images.githubusercontent.com/21338691/187735351-d0da89eb-3fdf-45c1-acc2-be886fbfb1e3.png)

## Detailed Description

`owner` field in Media collection.

```js
    {
      name: "owner",
      type: "relationship",
      relationTo: "admins",
      hasMany: false,
      required: true,
      defaultValue: ({ user }) => user.id,
      validate: (val, { user }) => {
        if (checkRole(["admin"], user)) {
          return true;
        }

        if (val) {
          if (val === user.id) {
            return true;
          }
        }
        return "User doesn't match the current user.";
      },
      admin: {
        readOnly: true,
      },
    },
```
browser: firefox
payloadCMS: 1.0.31

**Comment by DanRibbens** [comment]
I am able to reproduce this issue and will work on the fix for the next release. @MinSomai Thanks for reporting!

**Comment by DanRibbens** [comment]
Fixed in release v1.0.33
